packages/cxl-ui/src/components/cxl-icon-nav-item.js

```diff
@@ -0,0 +1,32 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import '@vaadin/vaadin-item';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlIconNavItemStyles from '../styles/cxl-icon-nav-item-css.js';
+
+class CXLIconNavItemElement extends LitElement {
+  static get properties() {
+    return {
+      href: {
+        type: String
+      },
+      icon: {
+        type: String
+      }
+    };
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlIconNavItemStyles];
+  }
+
+  render() {
+    return html`
+      <a href="${this.icon}">
```

Comment:
@chanar ouch, this has to be `${this.href}`

I'd actually prefer this to be fetched from element content. `<cxl-icon-nav-item href="#">cxl:training</cxl-icon-nav-item>`. Are there any drawbacks?

Reply:
`${this.innerHTML}` seems to work without side-effects.

Reply:
`href` attribute needs to be conditional. Not always guaranteed to be there (admin menu editor allows empty), and we should not render `undefined` values.

`<a ?href="${this.innerHTML}">`

Reply:
https://github.com/Polymer/lit-html/issues/625 actually points out `ifDefined()` is the correct solution.